### PR TITLE
Fixed javascript assignment expressions order and a 'sed' script

### PR DESCRIPTION
**Pull request by yanmingsohu**
The README of the 'sed' script needs to be updated, but I can't speak English well.

### CODE REVIEW

File: javascript/Cpp/pitch.sed [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#
+# https://github.com/antlr/grammars-v4/blob/master/javascript/Cpp/README.md
+#
+# sed -f pitch.sed JavaScriptParser.g4 > JavaScriptParser-CppRuntime.g4
+# sed -f pitch.sed JavaScriptLexer.g4  > JavaScriptLexer-CppRuntime.g4
+#
+# Replease 'this.' TO 'this->';
```
Comment (KvanTTT) [inline javascript/Cpp/pitch.sed]: Typo: `Replease` -> `Replace`.

File: javascript/Cpp/pitch.sed [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+/}/ {
+  z
+  h
+  d
+}
+
+/.*/ {
+  H
+}
```
Comment (KvanTTT) [inline javascript/Cpp/pitch.sed]: Is it a universal script for targeting .g4 files to C++ runtime? Can we place this file to the root directory and add info to README how to use it? As I understand the similar info is described in https://github.com/antlr/grammars-v4/tree/master/javascript/Cpp

@mike-lischke what do you think about it?

Comment (studentmain) [reply javascript/Cpp/pitch.sed]: > Is it a universal script for targeting .g4 files to C++ runtime?

Seems not yet, as there's hard coded #include, but maybe there's a chance to make it universal?

Comment (mike-lischke) [reply javascript/Cpp/pitch.sed]: Looks like this is some local stuff for the Javascript grammar with a C++ target. Nothing that should go into a global scope.

Comment (KvanTTT) [reply javascript/Cpp/pitch.sed]: Ok, I'm leaving it as is until better times.

**Comment by teverett** [comment]
@yanmingsohu  thanks

**Pull request merged**